README.md

```diff
@@ -1 +1,3 @@
-# RoboCup_RescueMaze_2020
\ No newline at end of file
+# RoboCup_RescueMaze_2020
+
+#By Emerico n' Marlon
```

Comment:
I'd suggest to also add the other team members.

Reply:
Ok, that was a test because I don't know why I was not able to make git push, so Aurora helped me and made that change to try to find the problem.